### PR TITLE
add r-plsrglm

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dbast** [comment]
Please recreate via https://github.com/bgruening/conda_r_skeleton_helper and fix spdx.

File: recipes/r-plsrglm/build.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -o errexit -o pipefail
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
+  ${R} CMD INSTALL --build .
```
Comment (dbast) [inline recipes/r-plsrglm/build.sh:7]:
```suggestion
  export DISABLE_AUTOBREW=1
  ${R} CMD INSTALL --build .
```

File: recipes/r-plsrglm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  version: {{ version|replace("-", "_") }}
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/plsRglm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/plsRglm/plsRglm_{{ version }}.tar.gz
+  sha256: eb98f250a75876e6bbedf245eb6c60cf48ba4775ec16b1f87b2f93c1c8489d00
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:16]:
```suggestion
```

File: recipes/r-plsrglm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  number: 0
+  # no skip
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:18]:
```suggestion
```

File: recipes/r-plsrglm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  noarch: generic
+  # This is required to make R link correctly on Linux.
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:20]:
```suggestion
```

File: recipes/r-plsrglm/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: plsdof, R.rsp, chemometrics, plsdepot
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:25]:
```suggestion
```

File: recipes/r-plsrglm/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  run:
+    - r-base
+    - r-mass
+    - r-bipartite
+    - r-boot
+    - r-car
+    - r-mvtnorm
+test:
+  commands:
+    # You can put additional test commands to be run here.
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:45]:
```suggestion
```

File: recipes/r-plsrglm/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    - $R -e "library('plsRglm')"           # [not win]
+    - "\"%R%\" -e \"library('plsRglm')\""  # [win]
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+  # requires:
+    # Put any additional test requirements here.
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:51]:
```suggestion
```

**Comment by dbast** [comment]
this no also contains r-prim

**Comment by FernandezMathieu** [comment]
Yep, actually struggling with my branches. Will be fixed in the hour.

**Comment by FernandezMathieu** [comment]
Can we merge now ? @dbast

File: recipes/r-plsrglm/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    - r-car
+    - r-mvtnorm
+test:
+  commands:
+    - $R -e "library('plsRglm')"           # [not win]
+    - "\"%R%\" -e \"library('plsRglm')\""  # [win]
+
+about:
+  home: http://www-irma.u-strasbg.fr/~fbertran/, https://github.com/fbertran/plsRglm
+  license: GPL-3.0-or-later
```
Comment (dbast) [inline recipes/r-plsrglm/meta.yaml:46]:
```suggestion
  license: GPL-3.0-only
```

**Pull request merged**